cpp/src/arrow/flight/sql/odbc/odbcabstraction/include/odbcabstraction/odbc_impl/odbc_connection.h

```diff
@@ -41,13 +41,21 @@ class ODBCConnection : public ODBCHandle<ODBCConnection> {
   ODBCConnection(const ODBCConnection&) = delete;
   ODBCConnection& operator=(const ODBCConnection&) = delete;
 
+  /// \brief Constructor for ODBCConnection.
+  /// \param environment[in] the parent environment.
+  /// \param spiConnection[in] the underlying spi connection.
   ODBCConnection(ODBCEnvironment& environment,
                  std::shared_ptr<driver::odbcabstraction::Connection> spiConnection);
 
   driver::odbcabstraction::Diagnostics& GetDiagnostics_Impl();
 
   const std::string& GetDSN() const;
   bool isConnected() const;
+
+  /// \brief Connect to Arrow server.
+  /// \param dsn[in] the dsn name.
+  /// \param properties[in] the connection property map extracted from connection string.
+  /// \param missing_properties[out??] -AL- todo find out what missing property is for? Seems to be a return param.
```

Comment:
The missing_properties map is for reporting in the OutConnectionString what properties are missing. It relates to what fields are supposed to be editable in the dialog and also iterative connection in SQLBrowseConnect.

Reply:
That's my understanding as well, thanks